tailwind.config.ts

```diff
@@ -22,6 +23,15 @@ export default {
 			fontFamily: {
 				sans: ["Kokoro", "sans-serif"],
 				mono: ["Jetbrains Mono"]
+			},
+			keyframes: {
+				float: {
+					"0%, 100%": { transform: "translateY(0)" },
+					"50%": { transform: "translateY(-10px)" }
+				}
+			},
+			animation: {
+				float: "float 3s ease-in-out infinite"
```

Comment:
Should we do these sort of animations?

I think it will make the future refractoring confusing.

Lets keep styles local for now?